src/scss/attendee/layer.scss

```diff
@@ -11,7 +11,7 @@
 // $z-index--chat: above($z-index--header);
 
 $-z-index--baseline: 1000;
-$z-index--baseline: $-z-index--baseline;
+$z-index--attendee: above($-z-index--baseline);
```

Comment:
The idea is to never have the baseline accessible from the outside, behaving like an anchor, or a singleton. All other layers can be relatively defined basted on it or on each other, having their own semantic names.

Please do note, public constants like `z-index--attendee` have only meaning and use if they are used to coordinate some baseline between different components, otherwise, for intra-component z-index values, module private constants should be just fine.